samples/java-fibonacci-action-pojo/src/main/java/com/example/Main.java

```diff
@@ -0,0 +1,28 @@
+/* This code was generated by Akka Serverless tooling.
+ * As long as this file exists it will not be re-generated.
+ * You are free to make changes to this file.
+ */
+
+package com.example;
+
+import com.akkaserverless.javasdk.AkkaServerless;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import com.example.fibonacci.FibonacciAction;
+import com.akkaserverless.javasdk.action.ActionPojoProvider;
+
+public final class Main {
+
+  private static final Logger LOG = LoggerFactory.getLogger(Main.class);
+
+  public static AkkaServerless createAkkaServerless() {
+    AkkaServerless akkaServerless = new AkkaServerless();
+    akkaServerless.register(ActionPojoProvider.of(FibonacciAction.class,__ -> new FibonacciAction()));
```

Comment:
Registering is a bit clunky, but we can improve it later. We might find something.